js/src/analysis/classicEditorData.js

```diff
@@ -102,18 +104,26 @@ class ClassicEditorData {
 	subscribeToElements() {
 		this.subscribeToInputElement( "title", "title" );
 		this.subscribeToInputElement( "excerpt", "excerpt" );
+		this.subscribeToInputElement( "excerpt", "excerpt", false );
```

Comment:
Shouldn't the forReplaceVariable of title be false as well? based on the switch statement in subscribeToElement?

Reply:
Rereading this after the weekend, it seems very ugly indeed 😅 , refactored it.